week1/Vectors.ipynb

```diff
@@ -0,0 +1,762 @@
+{
```

Comment:
<p>Also, talk about the distances you have mentioned in short and their applications.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/toggle-corp/ml-internship/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>